frog/imports/ui/GraphEditor/SidePanel/UploadFile.jsx

```diff
@@ -0,0 +1,55 @@
+// @flow
+
+import React, { Component } from 'react';
+import Form from 'react-jsonschema-form';
+
+class UploadFile extends Component {
```

Comment:
This class does not upload files. It converts files to dataURI. The file are uploaded only when we copy this dataURI in the configuration.